package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-file-upload",
-  "version": "2.6.1",
+  "version": "2.6.2",
```

Comment:
Typically the version bump is left up to the maintainer's discretion to bump. In case they want to release this along with a feature or what have you.

Reply:
I've seen it  both ways. I've been asked to bump on minor PRs. I agree etiquette would say not bumping - reversed